denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -242,29 +242,33 @@ def _create_zapper_imgmap(self, event):
                 )
             ]
         )
-        line_user = LineUser.objects.get(user_id=event.source.user_id)
-        if line_user.zapper_id:
-            # Use slice to prepend these three object to actions list.
-            zapper_imgmap.actions[:0] = [
-                URIImagemapAction(
-                    link_uri='https://example.com/{id}'.format(id=line_user.zapper_id),
-                    area=ImagemapArea(
-                        x=520, y=520, width=520, height=520
-                    )
-                ),
-                MessageImagemapAction(
-                    text='我想了解整個商圈的蚊蟲情況',
-                    area=ImagemapArea(
-                        x=520, y=0, width=520, height=520
-                    )
-                ),
-                MessageImagemapAction(
-                    text='我的補蚊燈需要專人協助',
-                    area=ImagemapArea(
-                        x=0, y=520, width=520, height=520
+        try:
+            line_user = LineUser.objects.get(user_id=event.source.user_id)
+        except LineUser.DoesNotExist:
+            logger.error('Line User Does Not Exist')
```

Comment:
Oh, I think it would be even better if you create a utility function for adding this row to the database if such exception occurs.
But I'll approve this PR. This could be an improvement.